build_common.ps1

```diff
@@ -1087,8 +1092,16 @@ class ModAssetsCookStep {
 				catch {
 					FailureMessage "Failed to restore previous $($this.cookerOutputPath)"
 					FailureMessage $_
+
+					$cleanupFailed = $true
 				}
 			}
+
+			if ($cleanupFailed) {
+				Write-Host ""
+				Write-Host ""
+				ThrowFailure "Failed to clean up the asset cooking hacks - your SDK is now in a corrupted state. Please preform the cleanup manually before building a mod or opening the editor."
```

Comment:
s/preform/perform. Also "the cleanup" is somewhat unhelpful because users would have to delve into the source code to figure out what FS operations they'd have to perform.

Reply:
> users would have to delve into the source code to figure out what FS operations they'd have to perform

Why? It says in the output above what operations failed. But yes, the user would need to understand how the process works, I'm not sure there is a way around that in case the automated logic has failed

Reply:
Ah right, I forgot that a failed cleanup step still tries to perform the other cleanups as well. Seems fine to me then, maybe `clean.ps1` can be enhanced to perform this cleanup too but it doesn't seem urgent.